pkg/kubectl/cmd/get/get.go

```diff
@@ -687,7 +691,15 @@ func (o *GetOptions) watch(f cmdutil.Factory, cmd *cobra.Command, args []string)
 	defer cancel()
 	intr := interrupt.New(nil, cancel)
 	intr.Run(func() error {
-		_, err := watchtools.UntilWithoutRetry(ctx, w, func(e watch.Event) (bool, error) {
```

Comment:
switching this to Until or UntilWithSync would handle connection interruptions, and is preferable to depending on a kept-alive connection. See https://github.com/kubernetes/kubernetes/pull/74717

Reply:
I will sync with that PR for the `get` case, thank you. Thoughts on the general approach of the PR?